src/allmydata/scripts/cli.py

```diff
@@ -224,7 +224,7 @@ class CpOptions(FileStoreOptions):
     def parseArgs(self, *args):
         if len(args) < 2:
             raise usage.UsageError("cp requires at least two arguments")
-        self.sources = map(argv_to_unicode, args[:-1])
+        self.sources = list(map(argv_to_unicode, args[:-1]))
```

Comment:
Maybe I missed a decision somewhere, but could these be expressed as list-comprehensions instead? (Or did we decide stuff like that is "too much" change?)

`[argv_to_unicode(arg) for arg in args[:-1]]` is what I'm thinking here..

Reply:
Yeah that's nicer.